src/OrchardCore/OrchardCore.Contents.Core/ContentsAdminListFilterOptions.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+
+namespace OrchardCore.Contents;
+
+public class ContentsAdminListFilterOptions
+{
+    public const string DefaultTermName = "text";
+
+    private readonly Dictionary<string, string> _defaultTermNames = new(StringComparer.OrdinalIgnoreCase);
+
+    /// <summary>
+    /// When true, the entire searched terms will be enclose within quotation marks, it creates an exact match search.
+    /// This means that search engines will treat the entire term as a single keyword or phrase, rather than individual words.
+    /// </summary>
+    public bool UseQuotationMarks { get; set; }
+
+    /// <summary>
+    /// Gets the term-name to use by the given content type.
```

Comment:
`term-name` => `term name` as in other places, maybe `default term name`

Reply:
done